### PR TITLE
Adding FSA package

### PR DESCRIPTION
**Pull request by Zyg-D**
Helper script was used: https://github.com/bgruening/conda_r_skeleton_helper

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by isuruf** [comment]
@Zyg-D, please don't add the pinning phrase in the top comment.

**Comment by dopplershift** [comment]
Also, you can update this PR by pushing new commits to your branch, you don't need to close the PR and open a new one.

**Comment by Zyg-D** [comment]
Hello, @conda-forge-admin, it seems now this can be merged.

**Comment by Zyg-D** [comment]
Hello, @conda-forge-admin, can this be merged?

**Comment by Zyg-D** [comment]
Hello, @dopplershift. Excuse me. Who should I ping in order to check and merge this PR?

**Comment by Zyg-D** [comment]
Also https://github.com/conda-forge/staged-recipes/pull/16195

**Comment by dopplershift** [comment]
Ping @conda-forge/help-r

File: recipes/r-fsa/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - "\"%R%\" -e \"library('FSA')\""  # [win]
+
+about:
+  home: https://github.com/droglenc/FSA
+  license: GPL-2.0-or-later
+  summary: A variety of simple fish stock assessment methods. Detailed vignettes are available
+    on the fishR website <http://derekogle.com/fishR/>.
+  license_family: GPL2
+  license_file:
+    - LICENSE
```
Comment (cbrueffer) [inline recipes/r-fsa/meta.yaml:58]:
```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
```

Comment (cbrueffer) [reply recipes/r-fsa/meta.yaml:58]: This uses the `r-base` installed licensed file. Please delete the LICENSE file you added in the PR.

**Review by cbrueffer — COMMENTED** [review]
Looks good, please just remove the LICENSE file and apply the suggestion.

**Comment by Zyg-D** [comment]
@cbrueffer - Thank you for the review. I've done what's required.

**Comment by cbrueffer** [comment]
Great, thanks!

**Pull request merged**